openshift/kustomize/services/reporting/base/deploy.yaml

```diff
@@ -154,6 +154,12 @@ spec:
                   name: ches
                   key: PASSWORD
 
+            - name: Charts__Url
```

Comment:
Fixed configuration for Reporting Service so that it can communicate with the Charts API.